### ISSUE TITLE
`MastForest`: Add convenience method for tests

### ISSUE THREAD

**Issue by plafer** [0xPolygonMiden/miden-vm#1355]
_Originally posted by @bobbinth in https://github.com/0xPolygonMiden/miden-vm/pull/1349#discussion_r1640372101_          
          
Not for this PR, but I see this pattern a lot in tests. I wonder if it makes sense to create convenience methods on `MastForest` which would create a node and add it in a single call. For example, something like:
```Rust
let last_basic_block_id = mast_forest.ensure_basic_block(vec![Operation::FmpAdd]);
```

**Comment by sergerad** [comment]
Happy to do this one 👍

**Comment by bobbinth** [comment]
Closed by #1404.
